crates/bitwarden-auth/src/common/enums/grant_type.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+
+/// Represents the OAuth 2.0 grant types recognized by the Bitwarden API.
+/// A grant type specifies the method a client uses to obtain an access token,
+/// as defined in [RFC 6749, Section 4](https://datatracker.ietf.org/doc/html/rfc6749#section-4)
+/// or by custom Bitwarden extensions. The value is sent in the `grant_type` parameter
+/// of a token request.
+#[derive(Serialize, Deserialize, Debug)]
+/// Instructs deserialization to map the string "send_access" to the `SendAccess` variant.
```

Comment:
Note that this doc comment applies to the `GrantType` struct, so the documentation might read a bit weird, as it will just add it to the end of the previous comment:

<img width="1003" height="220" alt="Image" src="https://github.com/user-attachments/assets/ed13f980-def1-4999-a5de-580e84a6ea59" />

We should change the comment by `//` or remove it.

Reply:
Oh, good catch! Resolved with 22d7846fd59aee0d648fbfb5395670505b229d8b